mtholyoke/docker-data-science-environment
# Fix `data/` directory mapping

## Scope

Put the `data/` directory into the container in a way that is accessible from the JupyterLab UI left rail to address #5.


## Implementation 

In the Dockerfile, added a new directory in the container called `jupyterlab` and symlinked the `analysis/` and `data/` directories into it. Then, set JupyterLab to use the new `jupyterlab` directory as the home directory.


## Future Work

Not at this time.



## Testing

1. From the project directory, check out this branch.
1. Open JupyterLab in the browser OR reload JupyterLab in the browser.
1. Observe that the `data/` and `analysis/` directories exist at the top level in the left sidebar.


### To be completed by the person testing

- [x] `analysis/` directory is in the left sidebar of the JupyterLab UI
- [x] `data/` directory is in the left sidebar of the JupyterLab UI


## Special considerations for making this live
- Will need to evaluate whether the [directions for syncing to the parent repository](https://github.com/mtholyoke/docker-data-science-environment/wiki/Advanced-topic-for-faculty-lead-researchers:-Working-with-the-parent-template-repository) will work for the forks being developed.
